tests/spec/core/markdown-spec.js

```diff
@@ -61,7 +61,7 @@ describe("Core - Markdown", function() {
     ops.config.format = "markdown";
     const doc = await makeRSDoc(ops);
     expect(doc.querySelector("code")).toBeFalsy();
-    expect(doc.querySelector("#foo").textContent === "Foo");
+    expect(doc.getElementById("foo").textContent).toBe("1. Foo");
```

Comment:
@marcoscaceres, is this expected?

Reply:
It looks correct... which is odd, considering the old test.

Reply:
I think the old test simply didn't work.

Reply:
Ah, right. It just “expect()”s nothing.